### PR TITLE
v1.16 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+## 1.16.0 Release
+- Release Date: January 16, 2019
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Combatibility
+
+ - Mobile App v1.13+ is required for Mattermost Server v5.4+.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+
+### Bug Fixes
```
Comment (esethna) [inline CHANGELOG.md]: We should sort these in rough order of severity in my mind (at least for the more severe bugs), ie...:

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+
+ - Fixed an issue where date separators did not update when changing timezones.
+ - Removed an extra divider below "Edit Channel" of Direct Message Channel Info.
+ - Fixed an issue where a quick double tap on switch of Channel Info created and extra on/off state.
+ - Fixed an issue where iOS long press menu didn't have rounded corners.
+ - Fixed an issue where the Favorites section did not clear from a users's channel drawer.
+ - Fixed an issue where a user was not returned to previously viewed channel after viewing and then closing an archived channel.
+ - Fixed an issue where the status changes for other users did not always stay current in the mobile app.
```
Comment (esethna) [inline CHANGELOG.md]: 1st

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,25 @@
...
+ - Fixed an issue where a post did not fail properly when the user attempted to send the post while there was no network access.
```
Comment (esethna) [inline CHANGELOG.md]: 2nd

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,25 @@
...
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Combatibility
+
+ - Mobile App v1.13+ is required for Mattermost Server v5.4+.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+
+### Bug Fixes
+
+ - Fixed an issue where date separators did not update when changing timezones.
```
Comment (esethna) [inline CHANGELOG.md]: 3rd

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,25 @@
...
+ - Removed an extra divider below "Edit Channel" of Direct Message Channel Info.
+ - Fixed an issue where a quick double tap on switch of Channel Info created and extra on/off state.
+ - Fixed an issue where iOS long press menu didn't have rounded corners.
+ - Fixed an issue where the Favorites section did not clear from a users's channel drawer.
```
Comment (esethna) [inline CHANGELOG.md]: 4th

**Comment by amyblais** [comment]
@enahum Can you help merge this PR?

**Pull request merged**